packages/datagrid/src/components/DefaultHeaderRenderer/DefaultHeaderRenderer.scss

```diff
@@ -10,6 +10,18 @@ $td-grip-resize-width: 6px !default;
 	}
 }
 
+.td-header-type {
+	&::first-letter {
+		text-transform: uppercase;
+	}
+	text-transform: lowercase;
```

Comment:
`text-transform: capitalize;` instead of first-letter + the rest

Reply:
we don't want My Subtitle but My subtitle

Reply:
Can you give me an example of types that are a group of words ?